tests/unit/test-poolman.cpp

```diff
@@ -20,16 +20,15 @@
 #define JERRY_MEM_POOL_INTERNAL
 
 #include "mem-allocator.h"
-#include "mem-pool.h"
 #include "mem-poolman.h"
 
 #include "test-common.h"
 
 // Iterations count
-const uint32_t test_iters = 16384;
+const uint32_t test_iters = 1024;
 
 // Subiterations count
-const uint32_t test_max_sub_iters = 32;
+const uint32_t test_max_sub_iters = 1024;
```

Comment:
Yes, it is ok. We change sub-iterations number from 32 to 1024 to increase testing coverage.